packages/tools/addon/styles/cs-field-overlays.css

```diff
@@ -1,20 +1,31 @@
-.cs-field-overlay .overlay-label {
-    font-size: 10px;
-    text-transform: uppercase;
-    background-color: var(--tools-highlight);
+.cs-field-overlay .overlay-label,
+.cs-content-overlay .overlay-label {
+  font-size: 10px;
+  text-transform: uppercase;
+  background-color: var(--deep-tools-highlight);
 }
 
-.cs-field-overlay .overlay-border {
-    border-color: var(--tools-highlight);
+.cs-field-overlay.with-inline-editor .overlay-label,
+.cs-field-overlay.passthrough .overlay-label {
+  background-color: var(--tools-highlight);
 }
 
-.cs-content-overlay .overlay-label {
-    font-size: 10px;
-    text-transform: uppercase;
-    background-color: var(--deep-tools-highlight);
+.cs-field-overlay .overlay-border,
+.cs-content-overlay .overlay-border {
+  border-color: var(--deep-tools-highlight);
+  background-color: var(--faded-tools-highlight);
 }
 
-.cs-content-overlay .overlay-border {
-    border-color: var(--deep-tools-highlight);
-    background-color: var(--faded-tools-highlight);
+.cs-field-overlay.with-inline-editor .overlay-border,
+.cs-field-overlay.passthrough .overlay-border {
+  border-color: var(--tools-highlight);
+  background-color: transparent;
+}
+
+.cs-field-overlay.passthrough {
+  pointer-events: none;
+}
+
+.cs-field-overlay.passthrough .overlay-scrim {
```

Comment:
@ef4 As the `.overlay-scrim`s  are children of the `cs-field-overlay` and we set `pointer-events: none` on the latter, I couldn't click on any of the scrims (=anywhere on screen) to dismiss the overlay since that css rules applied to them, too.

That's why I added this rule but I'm not wholly satisfied with it as it crosses boundaries between add-ons (`@cardstack/tools` and `ember-overlays`) and thus regressions are easier to introduce.

Reply:
Let's change this selector to `.cs-field-overlay.passthrough > *`. That is the actual intent, I think.

Reply:
The problem with that is that the overlay scrims (`.overlay-scrim`) are not direct descendants of the `.cs-field-overlay` because they are wrapped in a `liquid-if` (see [here](https://github.com/ef4/ember-overlays/blob/master/addon/templates/components/create-overlay.hbs#L16-L21))

Reply:
But I think pointer-events is inherited. So if we disable them for .cs-field-overlay and then re-enable them on each direct descendant, only `.cs-field-overlay` itself will have them disabled.